application/ledger_query_service/src/ethereum/queries.rs

```diff
@@ -347,58 +418,72 @@ mod tests {
             00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\
             0000000000000000000000000000";
 
+    const CONTRACT_ADDRESS: &str = "0xe46FB33e4DB653De84cB0E0E8b810A6c4cD39d59";
+    const REDEEM_LOG_MSG: &str =
+        "0xB8CAC300E37F03AD332E581DEA21B2F0B84EAAADC184A295FEF71E81F44A7413";
+    const RANDOM_LOG_MSG: &str =
+        "0xB8CAC300E37F03AD332E581DEA21B2F0B84EAAADC184A295FEF71E81F44A7412";
```

Comment:
:x: Can we have an actually different one here? :) Maybe rename it to `UNKNOWN_LOG_MESSAGE`?

Reply:
it is different and completely random :P 
`0xB8CAC300E37F03AD332E581DEA21B2F0B84EAAADC184A295FEF71E81F44A741`**`3`**
`0xB8CAC300E37F03AD332E581DEA21B2F0B84EAAADC184A295FEF71E81F44A741`**`2`**